newIDE/app/src/BehaviorsEditor/Editors/Physics2Editor/index.js

```diff
@@ -33,20 +40,40 @@ const NumericProperty = (props: {|
   onUpdate: (newValue: string) => void,
 |}) => {
   const { properties, propertyName, step, onUpdate } = props;
-
+  const property = properties.get(propertyName);
   return (
     <SemiControlledTextField
       fullWidth
-      value={properties.get(propertyName).getValue()}
+      value={property.getValue()}
       key={propertyName}
-      floatingLabelText={properties.get(propertyName).getLabel()}
+      floatingLabelText={property.getLabel()}
       step={step}
       onChange={onUpdate}
       type="number"
+      endAdornment={<UnitAdornment property={property} />}
     />
   );
 };
 
+const UnitAdornment = (props: {| property: gdPropertyDescriptor |}) => {
+  const { property } = props;
+  const measurementUnit = property.getMeasurementUnit();
+  return (
+    <Tooltip

```

Comment:
on the screenshot, it felt to me that the title has a bit too much top margin from the top of the tooltip.
Is it something than can be improved?

Reply:
We could add left and right margin I guess.